docs/docsource/operation/index.rst

```diff
@@ -1,5 +1,5 @@
 |product| Operation
-===================
+====================
```

Comment:
was this intentional? It doesn't line up anymore.

Reply:
That extra character was for the typo I deleted!